pkg/prefixcollector/collector.go

```diff
@@ -18,7 +18,7 @@
 package prefixcollector
 
 import (
-	"cmd-exclude-prefixes-k8s/internal/utils"
+	"cmd-exclude-prefixes-k8s/pkg/utils"
```

Comment:
1. Please move all changes that do not relate to this PR to a separate PR.
2. Why do we need to change internal to pkg?

Reply:
1. Other changes removed.
2. This would give us the opportunity to use this module in other module, but now I realize, that we can change it, if we'll really need it.